padle/padle.py

```diff
@@ -121,14 +126,41 @@ async def padleadmin(self, ctx):
     @padle.command()
     async def help(self, ctx):
         """Instructions for PADle"""
-        prefix = ctx.prefix
-        await ctx.send("- PADle is similar to Wordle, except for PAD cards.\n"
-                       "- You have infinite tries to guess the hidden PAD card (chosen from a list "
-                       "of more well-known monsters). Everyone is trying to guess the same PAD card. "
-                       "With each guess, you are given feedback as to how similar the two cards are, "
-                       "including comparing the awakenings, rarity, typings, attributes, and monster "
-                       "point sell value.\n- A new PADle is available every day. "
-                       "Use `{}padle start` to begin!".format(prefix))
+        await ctx.send(self._get_help_text(ctx))
+
+    def _get_help_text(self, ctx):
+        args = {"db": get_emoji("db"), "p": ctx.prefix, "tsubaki": get_emoji("tsubaki")}
+        help_text = """PADle is a Wordle-inspired guessing game, but with some differences!
```

Comment:
put these in a separate file called help_texts and import them from there imo, that'll make these easier to read & edit later